main.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import os
+
+import cv2
+
+import pipeline
+from video_rider import VideoReader
+
+
+def treadmill_coordinates(image):
+    camera1_coord_template = [(300 / 960, 0 / 480), (750 / 960, 450 / 480)]
```

Comment:
Should it be hardcoded?

Reply:
yes

Reply:
can't resolve, resolve please (no permissions most likely)